sunpy/map/sources/yohkoh.py

```diff
@@ -33,6 +33,18 @@ class SXTMap(GenericMap):
     Yohkoh was launched on 30 August 1991 and ceased operations on
     14 December 2001.
 
+    Notes
+    -----
+    Observer location:  We use the ITRS coordinates provided in
+    the FITS header (``LAT``, ``LONG``, ``RADIUS``)
+    for the spacecraft location even when coordinates in a
+    Heliographic frame are provided (``HGS_LON``, ``HGS_LAT``,``DSUN_OBS``).
```

Comment:
```suggestion
    Stonyhurst frame are provided (``HGS_LON``, ``HGS_LAT``, ``DSUN_OBS``).
```
I believe the lack of a space after the comma messed up the rendering

Reply:
I missed that!